tests/validation/test_transaction_structure.py

```diff
@@ -101,6 +101,15 @@ def test_create_tx_asset_type(create_tx):
     validate_raises(create_tx)
 
 
+def test_create_tx_no_asset_data(create_tx):
```

Comment:
No... But we dont have a test for every schematic violation. Also, a transaction with no ID will get an asset id mismatch error.